### ISSUE TITLE
[FEAT] Blacklisting drives

### ISSUE THREAD

**Issue by TechaNima** [AnalogJ/scrutiny#610]
Hi.

I am running Scrutiny on a TrueNAS Scale VM and as such it has a couple of virtual drives.
They are just the OS and Apps. Don't worry. Just to make it easier for backup and restore on the Proxmox side.

The problem is that Scrutiny picks them up and automatically fails them, since they obviously can't pass a SMART test or whatever Scrutiny tests. Now there is ofc the delete drive button, but it has never kept the drives gone past the next collector cron job.

I'd simply like to either have a settings tab where every drive shows up and can be toggled to show on the main page or not.
Or have the delete button actually delete the drives from Scrutiny. I'd prefer the former.

**Comment by dataprolet** [comment]
This is already possible via the collector.yaml configuration file:

https://github.com/AnalogJ/scrutiny/blob/master/example.collector.yaml#L38

**Comment by TechaNima** [comment]
> This is already possible via the collector.yaml configuration file:
> 
> https://github.com/AnalogJ/scrutiny/blob/master/example.collector.yaml#L38

Something like this really should be in the UI. 

How would I access the file? I'm running Scrutiny on TrueNAS Scale. So it's buried somewhere in the kube cluster.

**Comment by AnalogJ** [comment]
Unfortunately that's not possible. The web container does not have any access/knowledge of the collector containers - on purpose.

However, every collector.yaml config option can also be set via environmental variables

**Comment by TechaNima** [comment]
Then why not just have the web page have a setting to not display selected info? Doesn't matter to me if the info is collected or not. I just wanted a convenient way to remove it from the UI.

**Comment by dataprolet** [comment]
> Unfortunately that's not possible. The web container does not have any access/knowledge of the collector containers - on purpose.

Have you even read the comment? And maybe try to be less demanding of a FOSS-project that's maintained by a single volunteer.

**Comment by TechaNima** [comment]
> > Unfortunately that's not possible. The web container does not have any access/knowledge of the collector containers - on purpose.
> 
> Have you even read the comment? And maybe try to be less demanding of a FOSS-project that's maintained by a single volunteer.

Yes. I read the comment and suggested an alternative way of doing what I asked for. I don't feel like I'm asking for the world here. Not all of us want to learn how to interact with kube clusters for a simple toggle for info to be shown.

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
